examples/error-lifting/error_lifting.description

```diff
@@ -1,6 +1,3 @@
 // The error lifting operator enables you to navigate through a set of fields without worrying about 
-// error or nil values along the way. In an event of an error or a nil return, the operator stops
+// `error` or `nil` values along the way. In the event of an `error` or a `nil` return, the operator stops
 // the navigation and the value of the entire expression evaluates to `error`.
```

Comment:
Why are we talking about nil values in error lifting?

Reply:
Hmm would this be confusing? Also, that last sentence is wrong. Gives the impression that if it encounters an error or nil, the expression evaluates to error

Reply:
Updated to say `error` or `nil`